shared/wallets/wallet-list/wallet-row/index.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+import * as React from 'react'
+import {Box2, ClickableBox, Icon, Text, Avatar} from '../../../common-adapters'
+import {
+  globalStyles,
+  globalMargins,
+  globalColors,
+  isMobile,
+  platformStyles,
+  styleSheetCreate,
+} from '../../../styles'
+
+type Props = {
+  isSelected: boolean,
+  name: string,
+  keybaseUser: string,
+  contents: string,
+  onSelect: () => void,
+}
+
+const rightColumnStyle = platformStyles({
+  isElectron: {
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+  },
+})
+
+const rowHeight = isMobile ? 56 : 48
+
+const backgroundColor = globalColors.white
+const backgroundColorSelected = globalColors.blue
+
+const styles = styleSheetCreate({
+  avatar: {marginRight: globalMargins.xtiny},
+
+  container: {
+    backgroundColor,
+  },
+  containerSelected: {
+    backgroundColor: backgroundColorSelected,
+  },
+
+  containerBox: {
+    height: rowHeight,
+    backgroundColor,
+  },
+  containerBoxSelected: {
+    height: rowHeight,
+    backgroundColor: backgroundColorSelected,
+  },
+
+  rightColumn: rightColumnStyle,
+
+  title: {
+    ...globalStyles.fontSemibold,
+    ...rightColumnStyle,
+    color: globalColors.darkBlue,
+    backgroundColor,
+    fontSize: 13,
+  },
+  titleSelected: {
+    ...globalStyles.fontSemibold,
+    ...rightColumnStyle,
+    color: globalColors.white,
+    backgroundColor: backgroundColorSelected,
+    fontSize: 13,
+  },
+
+  amount: {
+    ...rightColumnStyle,
+    color: globalColors.black_40,
+    backgroundColor,
+    fontSize: 11,
+  },
+  amountSelected: {
+    ...rightColumnStyle,
+    color: globalColors.white,
+    backgroundColor: backgroundColorSelected,
+    fontSize: 11,
+  },
+})
+
+const WalletRow = (props: Props) => {
+  return (
+    <ClickableBox
+      onClick={props.onSelect}
+      style={props.isSelected ? styles.containerSelected : styles.container}
+    >
+      <Box2
+        style={props.isSelected ? styles.containerBoxSelected : styles.containerBox}
+        direction="horizontal"
+        fullWidth={true}
+      >
+        <Icon
+          type="icon-wallet-64"
+          color={globalColors.darkBlue}
+          style={{
```

Comment:
doesn't play well with the stylesheet, since Icon requires a stricter type than the stylesheet provides, since it excludes some keys

Reply:
Icon has a helper specifically for this purpose

Reply:
Nice, fixed.